designer/server/src/test/scala/pl/touk/nussknacker/ui/api/StatisticsApiHttpServiceBusinessSpec.scala

```diff
@@ -48,28 +51,35 @@ class StatisticsApiHttpServiceBusinessSpec
     with RestAssuredVerboseLoggingIfValidationFails
     with Eventually
     with MockitoSugar
-    with Matchers {
+    with Matchers
+    with WithDesignerConfig {
 
   override implicit def patienceConfig: PatienceConfig =
     PatienceConfig(timeout = Span(5, Seconds), interval = Span(0.5, Seconds))
 
-  private val nuVersion              = BuildInfo.version
-  private val questDbPath            = BetterFile.temp / "nu"
-  private val now                    = Instant.now()
-  private val yesterday              = now.plus(-1L, ChronoUnit.DAYS)
-  private val yesterdayPartitionName = DateTimeFormatter.ISO_LOCAL_DATE.format(yesterday.atZone(ZoneOffset.UTC))
-  private val statisticsNames        = StatisticName.values
-  private val statisticsNamesSize    = statisticsNames.size
-  private val statisticsByIndex      = statisticsNames.zipWithIndex.map(p => p._2 -> p._1).toMap
-  private val quote                  = '"'
-  private val random                 = new Random()
+  private val nuVersion                 = BuildInfo.version
+  private val questDbRelativePathString = s"it-tests/${UUID.randomUUID().toString}"
```

Comment:
where will it be placed? Because /tmp was a great choice because we don't have to think about cleaning it up